app/views/users/_form.html.erb

```diff
@@ -24,6 +24,10 @@
     <%= f.label :password %><br>
     <%= f.password_field :password %>
   </div>
+  <div class="field">
+    <%= f.label :password_confirmation %><br>
+    <%= f.password_field :password_confirmation %>
```

Comment:
does this work as it, or do we need to add :password_confirmation field to the user_params private method in user_controller.rb?

not all that familiar with confirmation fields.

Reply:
this works as it is, I'm pretty sure due to http://guides.rubyonrails.org/active_record_validations.html#confirmation "This validation creates a virtual attribute whose name is the name of the field that has to be confirmed with "_confirmation" appended."

Reply:
ok i see, nice.